### PR TITLE
React Native Flipper performance plugin

### PR DESCRIPTION
**Pull request by nabil-brn**
### 📝 Description

Implementation of [React Native Flipper performance monitor](https://github.com/bamlab/react-native-flipper-performance-monitor)

### ❓ Context

- **Impacted projects**: `` N/A
- **Linked resource(s)**: `` https://github.com/bamlab/react-native-flipper-performance-monitor#install-for-non-expo-projects

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo
https://user-images.githubusercontent.com/16896559/207097751-1c614659-3c7a-42b7-9e67-f81febb750d7.mov

### 🚀 Expectations to reach

Must work on Android and iOS

### CODE REVIEW

**Comment by nabil-brn** [comment]
✅  Tested on Android & iOS simulators

File: apps/ledger-live-mobile/ios/ledgerlivemobile/AppDelegate.m [hunk 1]
```diff
@@ -97,6 +98,7 @@ - (void) initializeFlipper:(UIApplication *)application {
     [client addPlugin: [[FKUserDefaultsPlugin alloc] initWithSuiteName:nil]];
     [client addPlugin: [FlipperKitReactPlugin new]];
     [client addPlugin: [[FlipperKitNetworkPlugin alloc] initWithNetworkAdapter:[SKIOSNetworkAdapter new]]];
+    [client addPlugin: [FlipperPerformancePlugin new]];
```
Comment (Justkant) [inline apps/ledger-live-mobile/ios/ledgerlivemobile/AppDelegate.m:101]: If I'm not wrong we're in 0.68.5 and the doc says to implement this part differently (https://github.com/bamlab/react-native-flipper-performance-monitor#-v0680)

**Review by Justkant — CHANGES_REQUESTED** [review]
I haven't tested yet with flipper locally but I think the docs says that we should implement the iOS part differently

File: apps/ledger-live-mobile/ios/ledgerlivemobile/AppDelegate.m (hunk 1 above)
Comment (nabil-brn) [reply apps/ledger-live-mobile/ios/ledgerlivemobile/AppDelegate.m:101]: Thanks @Justkant, I did, it's just that some lines existed before (ifdef, FlipperClient) so I added only the missing ones.

**Review by Justkant — APPROVED** [review]

**Comment by Justkant** [comment]
You just need to rebase and do a `pnpm i` to resolve the conflicts on the `pnpm-lock.yaml`

**Pull request merged**